test/utils/NavigatorBehaviour.scala

```diff
@@ -31,6 +38,12 @@ import scala.concurrent.ExecutionContext.Implicits.global
 trait NavigatorBehaviour extends PropertyChecks with OptionValues {
   this: WordSpec with MustMatchers =>
 
+  protected val emptyAnswers: UserAnswers = UserAnswers(Json.obj())
```

Comment:
This doesn't belong here. NavigatorBehaviour is a generic navigator test. Anything no-UK specific should be somewhere else, ie a non-UK specific test

Reply:
fixed